docs/atomic.md

```diff
@@ -0,0 +1,18 @@
+Atomic host bootstrap
+=====================
+
+Atomic host testing has been done with the network plugin flannel. Change the inventory var `kube_network_plugin: flannel`.
```

Comment:
It's ok to gradually introduce a new feature and eventually get to full support across all combinations, but we can't call it supported until it's covered by CI